packages/suite/src/components/suite/FormattedCryptoAmount.tsx

```diff
@@ -42,18 +42,19 @@ export const FormattedCryptoAmount = ({
     'data-test': dataTest,
     className,
 }: FormattedCryptoAmountProps) => {
-    const { CryptoAmountFormatter, SignValueFormatter, CurrencySymbolFormatter } = useFormatters();
+    const { CryptoAmountFormatter, SignValueFormatter, NetworkSymbolFormatter } = useFormatters();
 
     if (!value) {
         return null;
     }
 
     const networkSymbol = symbol?.toLowerCase() as NetworkSymbol;
-    const formattedSymbol = networkSymbol ? CurrencySymbolFormatter.format(networkSymbol) : '';
+    const formattedSymbol = networkSymbol ? NetworkSymbolFormatter.format(networkSymbol) : '';
     const formattedValue = CryptoAmountFormatter.format(value, {
         isBalance,
         symbol: networkSymbol,
         withSymbol: false,
+        isSatoshis: true,
```

Comment:
I am quite sure this could break desktop suite, because it's definitely not in sats everywhere in desktop suite. What you need is probably to use `!isBalance` here, but I am not sure about that, so please investigate and test that in desktop suite.

Reply:
In desktop suite, it's only used in wrapper component where I already made a change